gum-gum-htb.js

```diff
@@ -283,15 +326,15 @@ function GumGumHtb(configs) {
              * these local variables */
 
             /* the bid price for the given slot */
-            var bidPrice = curBid.price;
+            var bidPrice = curBid.ad.price;
```

Comment:
please indicate a pass when the price is undefined or curBid.ad is {}

Reply:
Should be taken care by the default values